runtime/syntax/unison.vim

```diff
@@ -0,0 +1,122 @@
+" Vim syntax file
+"
+" Language:        unison
+" Maintainer:      Anton Parkhomenko <anton@chuwy.me>
+" Last Change:     Jan 6, 2023
+" Original Author: John Williams, Paul Chiusano and Rúnar Bjarnason
+
+" Remove any old syntax stuff hanging around
+if version < 600
```

Comment:
This version check was removed from the distributed syntax files some years ago and can be reduced to a simple test for `b:current_syntax`.

See other distributed syntax files for an example.